nerdlets/shared/components/dimensionPicker/index.js

```diff
@@ -54,7 +54,7 @@ export default class DimensionPicker extends React.Component {
       <Dropdown
         spacingType={[Dropdown.SPACING_TYPE.MEDIUM]}
         title="Select CRM Attribute"
-        label="Please select your CRM attribute."
+        placeholder="Please select your CRM attribute."
```

Comment:
I don't see placeholder as an explicit prop for the dropdown component so I'm not sure this will actually do anything, we could remove it. Title is essentially the placeholder now

Reply:
will fix